supervisor/validate.py

```diff
@@ -148,7 +148,10 @@ def validate_repository(repository: str) -> str:
         vol.Optional(ATTR_IMAGE): docker_image,
         vol.Optional(
             ATTR_ADDONS_CUSTOM_LIST,
-            default=["https://github.com/hassio-addons/repository"],
+            default=[
+                "https://github.com/hassio-addons/repository",
+                "https://github.com/esphome/hassio",
```

Comment:
I have renamed the repository =)
```suggestion
                "https://github.com/esphome/home-assistant-addon",
```